WordPress/src/main/java/org/wordpress/android/ui/activitylog/list/filter/ActivityLogTypeFilterFragment.kt

```diff
@@ -65,6 +62,39 @@ class ActivityLogTypeFilterFragment : DialogFragment() {
         initAdapter()
     }
 
+    private fun initViewModel() {
+        viewModel = ViewModelProviders.of(this, viewModelFactory)
+                .get(ActivityLogTypeFilterViewModel::class.java)
+
+        viewModel.uiState.observe(viewLifecycleOwner, Observer { uiState ->
```

Comment:
Yes, I had the same thought that it'd be better to keep it for now.

Reply:
Hmmm... Is this due to compatibility reason with AS version or Kotlin versions? I am thinking the later, right? And if that is the case, I suggest having every develop be forced to update the AS Kotlin plugin to at least the Gradle's Kotlin version, currently `1.4.10`, as specified by our build files. This will make sure everyone is on the same boat. Wdyt?

Reply:
Good point! I thought it was the AS lint, but it's the kotlin plugin. In that case I agree with you.